cmd/ipfs/main_test.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
-	"testing"
-
 	"github.com/ipfs/go-ipfs/commands"
+
+	gc "gx/ipfs/QmPp8w6dCzx6vPNre8QVSbN7GCY9M7ornWesnrJg48vYxE/check.v1"
 )
 
-func TestIsCientErr(t *testing.T) {
-	t.Log("Catch both pointers and values")
-	if !isClientError(commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified value")
-	}
-	if !isClientError(&commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified pointer")
-	}
+func (m mainRun) TestIsCientErr(c *gc.C) {
```

Comment:
This new way of testing this is not shorter, cleaner, or easier to understand. I'm not on board with this.